hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/TestBlockScanner.java

```diff
@@ -290,6 +290,18 @@ public void testDisableVolumeScanner() throws Exception {
     }
   }
 
+  @Test(timeout=60000)
+  public void testDisableVolumeScanner2() throws Exception {
+    Configuration conf = new Configuration();
+    conf.setLong(DFS_BLOCK_SCANNER_VOLUME_BYTES_PER_SECOND, -1L);
+    TestContext ctx = new TestContext(conf, 1);
+    try {
+      Assert.assertFalse(ctx.datanode.getBlockScanner().isEnabled());
```

Comment:
There is already a static import for assertFalse, you don't need the prefix Assert.

Reply:
Thanks @ayushtkn  for your careful review, have just updated

Reply:
seems it's still not updated.

Reply:
Thanks @jojochuang  for you review, sorry for long time no reply, just have updated the comment.